Create Modals [LeafyGreen].sketch

sooachung opened the pull request:
Modals for Sketch Library! 
I tried my best but I think there might be a lot of changes that I need to make😬

sooachung committed:
Create Modals [LeafyGreen].sketch

audroue committed:
Testing my PR

DesignerDave reviewed (changes requested):
Hey @audroue!

I'm sure you've identified some of these issues already, but thought I'd take a quick look at this to note a few issues:

<img width="743" alt="Screen Shot 2020-08-19 at 1 43 46 PM" src="https://user-images.githubusercontent.com/25213299/90671142-46094700-e222-11ea-9ab4-70299d61d269.png">

- If you have a shorter amount of description text and resize the modal to the appropriate size, the title gets cut off.

<img width="633" alt="Screen Shot 2020-08-19 at 1 46 34 PM" src="https://user-images.githubusercontent.com/25213299/90671238-6a652380-e222-11ea-8598-4cb52440757c.png">

- Similar category of issue: the spacing between the description text box and the bottom action bar of the modal grows when you resize the modal vertically. Theres probably some pinning to the top/bottom to be done for both of these issues.

<img width="661" alt="Screen Shot 2020-08-19 at 1 49 41 PM" src="https://user-images.githubusercontent.com/25213299/90671536-dc3d6d00-e222-11ea-891b-7bb9dc8ebad4.png">

- The strong confirmation text input should probably be pinned to the bottom for ease of resizing.

- I'd love if we could add a base modal symbol for other modals designers make to use as well.

audroue commented:
Thanks for providing feedback on this! Looking into it now

audroue committed:
Updated Modals based on feedback

- Fixed all alignment issues (hopefully)
- Fixed overirdes
- Maybe some typography & spacing tweaks here and there

audroue committed:
Added Base Modal + Alignment/Fixes Updates

audroue committed:
Fixed pixel being off on marketing modal

DesignerDave reviewed (approved):
One typo I noticed: 
<img width="430" alt="Screen Shot 2020-08-26 at 4 47 32 PM" src="https://user-images.githubusercontent.com/25213299/91355212-da901e00-e7bb-11ea-8a6c-b8c0dc2a9637.png">

Aside from that, LGTM! Feel free to merge when you fix that issue.

audroue committed:
Fixed typo
